v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToSpanner.java

```diff
@@ -372,21 +375,6 @@ public interface Options extends PipelineOptions, StreamingOptions {
 
     void setTransformationContextFilePath(String value);
 
-    @TemplateParameter.Integer(
-        order = 22,
-        optional = true,
-        description = "Directory watch duration in minutes. Default: 10 minutes",
-        helpText =
-            "The Duration for which the pipeline should keep polling a directory in GCS. Datastream"
-                + "output files are arranged in a directory structure which depicts the timestamp "
-                + "of the event grouped by minutes. This parameter should be approximately equal to"
-                + "maximum delay which could occur between event occurring in source database and "
-                + "the same event being written to GCS by Datastream. 99.9 percentile = 10 minutes")
-    @Default.Integer(10)
-    Integer getDirectoryWatchDurationInMinutes();
-
-    void setDirectoryWatchDurationInMinutes(Integer value);
```

Comment:
Can you validate if we set this in SMT?  Even if its optional, if we are setting it, then removing it will cause SMT (and other customers using SMT) to break.

Reply:
I checked, we only do it in a unit test so no problems there